### PR TITLE
Adding reverse prop to anchor component so when applied to icon-label…

### PR DESCRIPTION
**Pull request by jwijay**
…/primary anchors, icons and labels are flipped. (For BC next/pagination footer.)

Signed-off-by: Jackie Wijaya <jackiewijaya@webmocha.com>

### CODE REVIEW

**Comment by ericsoderberghp** [comment]
What's the context where we need/want this?

**Comment by jwijay** [comment]
We're using anchors with icons+labels in the pagination footer, and sometimes we want an icon to be on the right side (e.g. Next -->, instead of --> Next) if we have <-- Previous and Next --> links.

File: src/js/components/Anchor.js [hunk 1]
```diff
@@ -52,20 +53,32 @@ export default class Anchor extends Component {
...
-        target={this.props.target}
-        onClick={this.props.onClick}
-        aria-label={this.props.a11yTitle}>
-        {icon}
-        {children}
-      </this.props.tag>
-    );
+    if (this.props.reverse) {
+      return (
+        <this.props.tag id={this.props.id} className={classes}
```
Comment (ericsoderberghp) [inline src/js/components/Anchor.js]: There's a fair amount of code duplication here. How about something like:

const first = (this.props.reverse ? children : icon);
const second = (this.props.reverse ? icon : children);

Comment (jwijay) [inline src/js/components/Anchor.js]: Thanks! That makes sense.

**Pull request merged**